### DISCUSSION TITLE
I'm still at 1.27.4. Are there any issues upgrading to 1.30.1 through a docker pull that I should know about? My Docker is already at V2.

### DISCUSSION THREAD

**Discussion by SylvainGa** [teslamate-org/teslamate#4134] [Q&A]
Nothing to add, it's all in the title of the post.

**Comment by cwanja** [comment]
Should be fine. Just backup your data. If you run into issues, you could have to incrementally upgrade to 'major' versions.

**Reply by SylvainGa** [reply]
Thanks. Any specific version of postgres I should be at?

**Reply by cwanja** [reply]
New installs are on 16 I think. If you feel comfortable, incrementally upgrade both that and TeslaMate.
